sites/mosul/configs/openmrs/initializer_config/ampathforms/F09-ITFC_Discharge_form.json

```diff
@@ -1060,7 +1060,7 @@
               }
             },
             {
-              "id": "numberOfDays",
+              "id": "numberOfDaysInIntensiveCare",
```

Comment:
Can we maintain the same format as we use in other places, if the ID is the same, we append it with section labels, for instance, `sectionLabelInCamelCase_questionId`?

I believe we should incorporate this change in the Form Generator as well.

Thanks!

Reply:
Actually we've used `sectionLabelInCamelCase_questionId` for most of those especially the `If Other Specify` fields. This makes sense especially when the the depedent field labels aren't related. And yes, we should update the script to do this as well.

For this particular one, i went with this route because it's more understandable as far as English language is concerned related to it's purpose. It's similar to the approach we used for the score fields.